platform/ui/src/components/NavBar/NavBar.mdx

```diff
@@ -25,6 +25,41 @@ import { NavBar } from '@ohif/ui';
   </NavBar>
 </Playground>
 
+
```

Comment:
You can remove the changes in this file, and then I think we're good for merge.
The `Study List` view captures these, and we discussed ditching `header.js`, so I'm all good.